models/evc.py

```diff
@@ -787,6 +801,7 @@ def setup_failover_path(self):
 
         reason = ""
         self.remove_path_flows(self.failover_path)
+        self.failover_path = Path([])
```

Comment:
reseting the failover_path here helps reducing the possibility of running into a new race condition for get_failover_paths() -- I didn't add the lock mechanism there to avoid impacting the fast handle link down (it can be necessary in the future)